### ISSUE TITLE
Berserkir Fleet

### ISSUE THREAD

**Issue by ExodistSKY1** [tsunamayo/Skywanderers-Pathfinder-Group-Build#22]
Small fighter, Drop ship, Corvette, Frigate var1, Frigate var2, Destroyer (166m)

![20200503121955_1](https://user-images.githubusercontent.com/45324484/80920969-de91bf00-8d38-11ea-869f-390cb0936029.jpg)
![20200503121057_1](https://user-images.githubusercontent.com/45324484/80920974-e2254600-8d38-11ea-98d4-974052965065.jpg)

Need to finish:

-interior on the larger 3 
-Turrets
-A larger fighter/interceptor

Possibly:
-A larger 300m carrier/battle-cruiser  
-Miner

I do not have the BP's currently due to the export bug

**Comment by tsunamayo** [comment]
@ExodistSKY1 Hi, I dont know if you had made progress on this fleet? They look great! For now the best would be to use turret as fixed mounted is not quite working. Thanks!

**Comment by ExodistSKY1** [comment]
@tsunamayo  Yes I have and this has turned into a team build with Majikmonster helping out with the interiors! 

Would you want me to make a 300m carrier/battle cruiser for the fleet or should I keep the destroyer the largest for this faction?

What other ships would you like to have made for the Faction?
